geode-core/src/main/java/org/apache/geode/internal/cache/RegionPerfStats.java

```diff
@@ -463,7 +491,7 @@ protected void incEventThreads(int items) {
 
   @Override
   public void incEntryCount(int delta) {
-    stats.incLong(entryCountId, delta);
+    entryCount.addAndGet(delta);
```

Comment:
Why are we removing the region perf stat for entry count?

Reply:
To avoid updating two adders on each call, and to localize (to the constructor) the eventual work of removing stats.

Reply:
We had this conversation on the mailing list though and thought the consensus was to leave stats in place just wrap with meters. This approach of moving them out of stats into something else and having meters and stats poll it was clearly not accepted in the discussions.

Reply:
See concerns in whole PR comment.